qa/tasks/salt_manager.py

```diff
@@ -132,6 +132,16 @@ def all_minions_zypper_status(self):
         self.all_minions_zypper_lu()
         self.all_minions_zypper_ps()
 
+    def all_minions_zypper_up_if_needed(self):
+        """
+        List updates and perform update if needed
+        """
+        zypper_lu = "sudo salt \\* cmd.run \'zypper lu || true\' 2>/dev/null"
+        zypper_up = "zypper -n up"
```

Comment:
will this run successfully even without sudo?

Reply:
No you are right my bad I missed it.